sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/implementation/ApplicationGatewayListenerImpl.java

```diff
@@ -248,17 +259,23 @@ public ApplicationGatewayListenerImpl withHttps() {
     @Override
     public ApplicationGatewayListenerImpl withHostname(String hostname) {
         this.innerModel().withHostname(null);
```

Comment:
Move this line into `if (hostname != null) {`?

Reply:
Moved, thanks...